src/patternfly/components/SimpleList/simple-list.scss

```diff
@@ -58,12 +62,35 @@
     --#{$simple-list}__item-link--BackgroundColor: var(--#{$simple-list}__item-link--hover--BackgroundColor);
     --#{$simple-list}__item-link--Color: var(--#{$simple-list}__item-link--hover--Color);
 
+    position:relative;
```

Comment:
Can you move this up to the base element so that it applies all the time, not just on `:hover/:focus`? That way, if someone wants to turn on this border for whatever reason in their own custom theme for a use case that isn't `:hover/:focus`, they can. You probably have to add it as the first rule here, above `display: block;` https://github.com/patternfly/patternfly/blob/ae70c4e89b6779ca4e02c3d05dbaca13b3252537/src/patternfly/components/SimpleList/simple-list.scss#L40-L41